src/ViewToggleButton/ViewToggleButton.js

```diff
@@ -0,0 +1,75 @@
+import * as React from "react";
+
+import { GridView, TableRowsOutlined } from "@mui/icons-material";
+import { ToggleButton, ToggleButtonGroup } from "@mui/material";
+
+import PropTypes from "prop-types";
+
+/**
+ * Horizontal alignment button group component to toggle between card/table view
+ */
+export default function ViewToggleButton({
+  disabled = false,
+  onChange = () => {},
+  orientation = "horizontal",
```

Comment:
I don't think we want to support orientation for this component.

Reply:
removed